client/app/lib/utils.js

```diff
@@ -51,7 +51,7 @@ export function localizeTime(time) {
     .utc()
     .hour(hrs)
     .minute(mins)
-    .local()
+    .locale()
```

Comment:
Drive-by to fix a frontend error discovered when making the system stats page work. Want in a different PR?

Reply:
@kapilt I removed the drive-by formatting, would you prefer this as a distinct PR?

Reply:
I'll put this in a different PR because it will be easier to submit the schema changes upstream that way.

Reply:
your call, sounds good.